### ISSUE TITLE
Wrong HTML Codes

### ISSUE THREAD

**Issue by 0lumide** [vdurmont/emoji-java#12]
Some emojis, are a combination of several emojis seperated by the zero width joiner \U200D, such as
```
family (man, man, boy)
family (man, man, girl)
family (man, man, girl, boy)
family (man, man, boy, boy)
family (man, man, girl, girl)
```
but when computing the HTML Codes, you use `Character.codePointAt(this.unicode, 0)` which gets only the first code point, and as a result all the examples above all have the same HTML code as the man emoji, which is `&#128104;`

**Comment by vdurmont** [comment]
Hmmm yep you're right! Feel free to submit a PR to fix this!

**Comment by vdurmont** [comment]
Fixed by #13

**Comment by vdurmont** [comment]
I released a new version 2.2.1 that includes your fix!

**Comment by 0lumide** [comment]
Awesome!
